test/e2e/kubeproxy.go

```diff
@@ -197,6 +197,10 @@ func (config *KubeProxyTestConfig) hitNodePort(epCount int) {
 	config.dialFromNode("udp", node2_IP, nodeUdpPort, tries, epCount)
 	By("dialing(http) node1 --> node2:nodeHttpPort")
 	config.dialFromNode("http", node2_IP, nodeHttpPort, tries, epCount)
+
+	By("checking kube-proxy URLs")
+	config.getSelfURL("/healthz", "ok")
```

Comment:
Given that this test does NO retries or readiness, I'd like to defer that
extra work and get this and other PRs in before freeze.

On Tue, Feb 2, 2016 at 1:36 PM, Prashanth B <notifications@github.com>
wrote:

> In test/e2e/kubeproxy.go
> <https://github.com/kubernetes/kubernetes/pull/20464#discussion_r51639186>
> :
>
> > @@ -197,6 +197,10 @@ func (config *KubeProxyTestConfig) hitNodePort(epCount int) {
> >  	config.dialFromNode("udp", node2_IP, nodeUdpPort, tries, epCount)
> >  	By("dialing(http) node1 --> node2:nodeHttpPort")
> >  	config.dialFromNode("http", node2_IP, nodeHttpPort, tries, epCount)
> > +
> > +	By("checking kube-proxy URLs")
> > +	config.getSelfURL("/healthz", "ok")
>
> you can poll for ready though, from the test. I *think* the reboot tests
> do this and if you don't have a probe the kubelet just starts you off as
> ready.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/20464/files#r51639186>.
>